app/components/ui/molecules/SearchBox.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import styled from 'styled-components'
+import Autosuggest from 'react-autosuggest'
+
+import Icon from '../atoms/Icon'
+import theme from './SearchBox.local.css'
+
+const Root = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 0px;
+  position: relative;
+`
+
+const IconContainer = styled.div`
+  position: absolute;
+  right: 10px;
+  display: flex;
+  align-items: center;
+  height: 100%;
+`
+
+class SearchBox extends React.Component {
+  state = {
+    value: '',
+    suggestions: [],
+  }
+  onChange = (_, { newValue }) => {
+    this.setState({
+      value: newValue,
+    })
+  }
+  onSuggestionsFetchRequested = ({ value }) => {
+    this.setState({
+      suggestions: this.getSuggestions(value),
+    })
+  }
+  onSuggestionsClearRequested = () => {
+    this.setState({
+      suggestions: [],
+    })
+  }
+
+  getSuggestions = value => {
+    const inputValue = value.trim().toLowerCase()
+    const inputLength = inputValue.length
+    return inputLength === 0
+      ? []
+      : this.props.options.filter(
+          option => option.value.toLowerCase().search(inputValue) >= 0,
```

Comment:
`String.prototype.search` converts the input to a regular expression which will have unexpected consequences if a search contains any special characters. Did you mean `String.prototype.includes`?

Reply:
Hm, didn't know about that. I want to know if `option.value.toLowerCase()` has substrings of `inputValue`, and you're right, [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) does the job, thanks!